unit-tests/tests.h

```diff
@@ -0,0 +1,104 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2015 Intel Corporation. All Rights Reserved.
+
+#pragma once
+#ifndef TESTS_H
```

Comment:
No need for both `#pragma` and this

Reply:
And why `tests.h` plural? Why not `test,h`?

Reply:
done